couchdb/server/compiler.py

```diff
@@ -78,3 +96,48 @@ def helper():
         'parent': mod,
         'id': (idx is not None) and (idx + '/' + name) or name
     })
+
+
+def import_b64egg(b64str, egg_cache=None):
+    """Imports top level namespace from base64 encoded egg file.
+
+    For Python 2.4 `setuptools <http://pypi.python.org/pypi/setuptools>`_
+    package required.
+
+    :param b64str: Base64 encoded egg file.
+    :type b64str: str
+
+    :return: Egg top level namespace or None if egg import disabled.
+    :rtype: dict
+    """
+    if iter_modules is None:
+        raise ImportError('No tools available to work with eggs.'
+                          ' Probably, setuptools package could solve'
+                          ' this problem.')
```

Comment:
Do you mean.... [this block](https://github.com/djc/couchdb-python/pull/286/commits/2f562a771cc8e6258649453088647f3e3c95eb43#diff-dc62f4dea2af2a84d46bbaaad1aefbdbR122)?